### ISSUE TITLE
Navigation bar should be the first one available

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1152]
![Image](https://github.com/cryptpad/cryptpad/assets/95225431/ae7b764a-fe58-4f0a-ba5a-2057fa623e15)

the navigation bar should be available first when trying to navigate with the keyboard, instead navigation starts with the `Contact` link.

Problem encountered on Chrome/Firefox.

**Comment by dariiing** [comment]
The issue occurs only when I enter the focus on the main content(basically clicking with the mouse on the page), so there is no actual bug to fix here. Tested on Chrome/Firefox/Edge.
